packages/header-component/src/components/dc-header/dc-dropdown.tsx

```diff
@@ -77,10 +77,9 @@ export class Dropdown {
           ref={(el) => this.dropdownItems = el}
         >
           {
-            this.items.map(item => (
-              <a href={item.href} class="dropdown-item">
+            this?.items?.map(item => (
+              <a href={item.href} class="dropdown-item" target="_blank">
```

Comment:
I will rather do what you have done at packages/header-component/src/components/dc-header/dc-header.tsx in fact even better I would choose to spread the attributes of the link so you will have like:

```
[{"href":"http://debtcollective.org/","text":"About us", "target": "_blank"}, {"href":"http://x.org/","text":"About us"}, {"href":"http://google.org/","text":"About us", "data-testid": "hello_world"}]
```

And in that case the extra attribute will be rendered without extra customisation.

Reply:
Nice idea, I've made an update to spread props in the anchors